tests/unit/python/foglamp/services/common/test_services_common_utils.py

```diff
@@ -0,0 +1,134 @@
+                url_ping = "{}://{}:{}/foglamp/service/ping".format(service._protocol, service._address, service._management_port)
+                log_params = 'Ping not received for Service %s id %s at url %s attempt_count %s', service._name, service._id, \
+                       url_ping, utils._MAX_ATTEMPTS+1
+                resp = await utils.ping_service(service, loop=loop)
+
+        # THEN ping response is NOT received
+        assert resp is False
+        log.assert_called_once_with(*log_params)
+
+    async def test_shutdown_service_pass(self, test_server, loop):
+        # GIVEN a service is running at a given URL
+        app = web.Application()
+        with patch.object(FoglampMicroservice, "__init__", return_value=None):
+            m = mServiceThing()
+            # fill route table
+            routes.setup(app, m)
+
+            server = await test_server(app)
+            server.start_server(loop=loop)
+
+            # WHEN shutdown call is made at the valid URL
+            with patch.object(utils._logger, "info") as log:
+                service = ServiceRecord("d", "test", "Southbound", "http", server.host, 1, server.port)
+                url_shutdown = "{}://{}:{}/foglamp/service/shutdown".format(service._protocol, service._address,
+                                                                            service._management_port)
+                log_params1 = "Shutting down the %s service %s ...", service._type, service._name
+                log_params2 = 'Service %s, id %s at url %s successfully shutdown', service._name, service._id, url_shutdown
+                resp = await utils.shutdown_service(service, loop=loop)
+
+        # THEN shutdown returns success
+        assert resp is True
+        log.assert_called_with(*log_params2)
+
+    async def test_shutdown_service_fail_bad_url(self, test_server, loop):
+        # GIVEN a service is running at a given URL
+        app = web.Application()
+        with patch.object(FoglampMicroservice, "__init__", return_value=None):
+            m = mServiceThing()
+            # fill route table
+            routes.setup(app, m)
+
+            server = await test_server(app)
+            server.start_server(loop=loop)
+
+            # WHEN shutdown call is made at the invalid URL
+            with patch.object(utils._logger, "info") as log1:
+                with patch.object(utils._logger, "exception") as log2:
+                    service = ServiceRecord("d", "test", "Southbound", "http", server.host, 1, server.port+1)
+                    log_params1 = "Shutting down the %s service %s ...", service._type, service._name
+                    log_params2 = 'Error in Service shutdown %s, %s', service._name, ' '
+                    resp = await utils.shutdown_service(service, loop=loop)
+
+        # THEN shutdown fails
+        assert resp is False
+        log1.assert_called_with(*log_params1)
+        log2.assert_called()
```

Comment:
indent above 3 lines,
`log1.assert_called_once_with(*log_params1)` and 
`log2.assert_called_once_with(*log_params2)`